packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -61,7 +62,7 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, ITooltipHostSt
         { ...{ onBlurCapture: this._onTooltipMouseLeave } }
         onMouseEnter={ this._onTooltipMouseEnter }
         onMouseLeave={ this._onTooltipMouseLeave }
-        aria-describedby={ isTooltipVisible ? tooltipId : undefined }
+        aria-describedby={ (isTooltipVisible && !stopAriaDescribedBy) ? tooltipId : undefined }
```

Comment:
@mdahamiwal i believe aria-describedby gets placed onto the element that receives focus